evm/utils.js

```diff
@@ -777,6 +779,57 @@ function getConfigByChainId(chainId, config) {
     throw new Error(`Chain with chainId ${chainId} not found in the config`);
 }
 
+function findProjectRoot(startDir) {
+    let currentDir = startDir;
+
+    while (currentDir !== path.parse(currentDir).root) {
+        const potentialPackageJson = path.join(currentDir, 'package.json');
+
+        if (fs.existsSync(potentialPackageJson)) {
+            return currentDir;
+        }
+
+        // Move up a directory
+        currentDir = path.resolve(currentDir, '..');
+    }
+
+    throw new Error('Unable to find project root');
+}
+
+function findContractPath(dir, contractName) {
+    const files = fs.readdirSync(dir);
+
+    for (const file of files) {
+        const filePath = path.join(dir, file);
+        const stat = fs.statSync(filePath);
+
+        if (stat && stat.isDirectory()) {
+            const recursivePath = findContractPath(filePath, contractName);
+            if (recursivePath) return recursivePath;
+        } else if (file === `${contractName}.json`) {
+            return filePath;
+        }
+    }
+}
+
+function getContractPath(contractName) {
+    const projectRoot = findProjectRoot(__dirname);
+
+    const searchDirs = [
+        path.join(projectRoot, 'node_modules', '@axelar-network/axelar-gmp-sdk-solidity/artifacts/contracts'),
```

Comment:
```suggestion
        path.join(projectRoot, 'node_modules', '@axelar-network', 'axelar-gmp-sdk-solidity', 'artifacts, 'contracts'),
```
should use full join to be os independent